source/shared/net/resources/glossary.rst

```diff
@@ -197,6 +211,13 @@ Decryption key
 Dual to :ref:`encryption key<glossary-encryption-key>`. In contrast to the encryption key, which is public,
 this key is only known to the account holder.
 
+.. _glossary-delegate:
+
+Delegate
```

Comment:
Possibly this should be "Delegator" or "Delegating". We don't use "Delegate" as a noun, and it seems confusing to use as a verb here.